Calculator.py

```diff
@@ -26,4 +29,4 @@ def main():
     print(run_operation(number_1, number_2, operation))
 
 if __name__ == "__main__":
-	main()
\ No newline at end of file
+    main()
```

Comment:
Could you put main back where it was in the correct position?

Reply:
I agree that indentation can be quite crucial.